axon-server-connector/src/main/java/org/axonframework/axonserver/connector/heartbeat/Scheduler.java

```diff
@@ -0,0 +1,96 @@
+package org.axonframework.axonserver.connector.heartbeat;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+
+/**
+ * Responsible to scheduler at fixed rate a generic task.
+ * The scheduling can be started an stopped at any time.
+ *
+ * @author Sara Pellegrini
+ * @since 4.2
+ */
+public class Scheduler {
+
+    private final Logger log = LoggerFactory.getLogger(Scheduler.class);
```

Comment:
The regular naming for a `Logger` instance is `logger`, and we always make them `private static final` in the framework.